addons/website/views/snippets.xml

```diff
@@ -1524,5 +1524,18 @@
             </we-collapse>
         </we-collapse-area>
     </div>
+
+    <!-- Stretch section -->
+    <div data-js="SectionStretch"
+        data-selector="section"
+        data-target="> .container, > .container-fluid">
+        <we-collapse-area>
+            <we-toggler><i class="fa fa-fw fa-arrows-h"/> Stretch section</we-toggler>
```

Comment:
Rename into Stretch.

Reply:
@sdegueldre You can make that change then we will merge :+1:

Reply:
Done